src/Gearer.php

```diff
@@ -33,34 +32,34 @@ public function setConfig($gatewayId, $gatewaySecret): self
 
     public function createOrder(float $amount, int $keychainId, array $callbackData = [])
     {
-        $url = $this->apiEndpoint('orders');
-
         $callbackData = array_merge(config('gearer.defaults.callback_data'), $callbackData);
 
-        return $this->makeRequest('POST', $url, [
-            'amount' => $this->formatAmount($amount),
-            'keychain_id' => $keychainId,
-            'callback_data' => json_encode($callbackData)
-        ]);
+        return $this
+            ->apiEndpoint('orders')
+            ->makeRequest('POST', [
+                'amount' => $this->formatAmount($amount),
+                'keychain_id' => $keychainId,
+                'callback_data' => json_encode($callbackData)
+            ]);
     }
 
-    public function cancelOrder($orderOrPaymentId) : bool
+    public function cancelOrder($orderOrPaymentId): bool
     {
-        $url = $this->apiEndpoint("orders/{$orderOrPaymentId}/cancel");
-
-        $response = $this->makeRequest('POST',$url);
+        $response = $this
+            ->apiEndpoint("orders/$orderOrPaymentId/cancel")
+            ->makeRequest('POST');
 
-        return $response == null;
+        return is_null($response);
     }
 
     public function checkOrderStatusManually($paymentId)
     {
-        $url = $this->apiEndpoint("orders/{$paymentId}");
-
-        return $this->makeRequest('GET', $url);
+        return $this
+            ->apiEndpoint("orders/$paymentId")
+            ->makeRequest('GET');
     }
 
-    public function handleOrderStatusCallback(?Request $request = null)
+    public function handleOrderStatusCallback(?Request $request = null): array
```

Comment:
I had changed return type to array to be more consistent, but if there is any other piece of software that depends on this as a library, this might be a breaking change. So need to evaluate.